recipes/dsrc/meta.yaml

```diff
@@ -1,33 +1,37 @@
+{% set name = "dsrc" %}
+{% set version = "2014.12.17" %}
```

Comment:
why is version set to a date instead of real version?

Reply:
I don't remember what was the rationale for that decision. I guess we can switch to 2.0.2 but I'm wondering what would be the effect if we switch to 2.0.2. I mean how the version 2014.12.17 would compare to 2.0.2 Unfortunately 2.0.2 is smaller... but the build number may help if we increase it.